drivers/entropy/Kconfig.nrf5

```diff
@@ -29,28 +29,30 @@ config ENTROPY_NRF5_THR_THRESHOLD
 	range 4 255
 	help
 	  Low-water threshold in bytes to trigger entropy generation for
-	  thread mode consumers.
+	  thread mode consumers. As soon as the currently available bytes go
+	  below this mark hardware entropy generation will be started.
 
 config ENTROPY_NRF5_ISR_THRESHOLD
 	int "ISR low-water threshold"
 	range 12 255
 	help
 	  Low-water threshold in bytes to trigger entropy generation for
-	  ISR consumers.
+	  ISR consumers. As soon as the currently available bytes go
+	  below this mark hardware entropy generation will be started.
 
 config ENTROPY_NRF5_THR_BUF_LEN
 	int "Thread-mode buffer length"
 	range ENTROPY_NRF5_THR_THRESHOLD 255
```

Comment:
Is this the max number of entropy bytes as well?

Reply:
No, the buffer length is the max number of bytes you can retrieve from ISR in one call